R/count_teleconnections.R

```diff
@@ -53,6 +57,10 @@ count_watershed_teleconnections <- function(data_dir,
   # read reclassified crop table
   reclassify_raster(crop_cover_levels = levels(cropcover_USA)[[1]]) -> crop_reclass_table
 
+  # read NID point file and select only Flood Control Dams (C = Flood Control)
+  import_shapefile(paste0(data_dir, dams_file_path)) %>%
+    subset(Purposes == "C") -> flood_control_dams
```

Comment:
Please check that this covers all cases of flood control. For example, some dams might have several purposes and be labelled "ABC". These would be filtered out in the above code. If this is the case then you'll want to do something like `subset(grepl("C", Purposes))`